### ISSUE TITLE
Is this project dead?

### ISSUE THREAD

**Issue by Arminmsg** [iptc/extra#1]
Came across extra, but it seems the last update was when you were looking for developers but nothing happened afterwards?

**Comment by iptc** [comment]
No, it has been finished a month ago and the "active" repositories are just across the corner:
https://github.com/iptc/extra-core
https://github.com/iptc/extra-ext
https://github.com/iptc/extra-examples

We will integrate them in September.

**Comment by AlJohri** [comment]
just checking in @iptc, what did you mean by integrating? is this in the works? thanks!

**Comment by stuartmyles** [comment]
It is not dead, yet. But there hasn't been much in the way of recent work.

We are hoping to get the work going again. And are keen to hear if there are people who would like to be involved. Or who have suggestions for the work.
